build/verify-test-urls.sh

```diff
@@ -17,28 +25,33 @@ https://github.com/a/REAL/url/here
 https://github.com/another/REAL/url/here
 
 */
-
-You can find or create a test URL on our sandbox repo: https://github.com/refined-github/sandbox
 "
 
 # Replace line breaks with "%0A"
 # https://github.com/actions/toolkit/issues/193#issuecomment-605394935
-ANNOTATION=${ANNOTATION//$'\n'/%0A}
+ERROR_MESSAGE=${ERROR_MESSAGE//$'\n'/%0A}
 
 echo WILL VERIFY:
 echo
 
 for FILE in "$@"; do
+	LAST_LINE=$(wc -l < "$FILE")
+
 	if grep -q "test url" -i "$FILE"; then
 		echo ✅ "$FILE"
+		echo "::notice file=$FILE,line=$LAST_LINE::✅" >> "$ANNOTATIONS"
```

Comment:
Highlight the test URLs on every file. This helps us review and test the features because now the URLs are right where we need them.


<img width="471" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/c8f7464a-6eb2-4040-b229-7e83366aad07">